index.bs

```diff
@@ -4149,7 +4162,11 @@ To <dfn>trigger aggregatable attribution</dfn> given an [=attribution trigger=]
 1. If the result of running [=check if attribution should be blocked by rate limits=]
     with |trigger|, |sourceToAttribute|, and |rateLimitRecord| is not null,
     return it.
-1. If |sourceToAttribute|'s [=attribution source/number of aggregatable attribution reports=] value is equal to [=max aggregatable reports per source=][0], then:
+1. If |sourceToAttribute|'s [=attribution source/max number of aggregatable reports=] is greater than
```

Comment:
Is there any privacy concern with checking this condition as soon as we have the attributed source? If not, I think it'd be a lot easier to understand that there are no side effects.

CC @linnan-github.

Reply:
It would also allow implementations to optimize by skipping the rate limits checks above if the trigger would be rejected anyway.

Reply:
Actually, if we have this at the end and get a debug report, wouldn't it reveal that other limits passed?

Reply:
Moved.